app/assets/stylesheets/components/_card_for_superpowers.scss

```diff
@@ -0,0 +1,40 @@
+.cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 16px;
+}
+
+// Smallest device
+@media (min-width: 100px) and (max-width: 575px) {
+  .cards {
+    grid-template-columns: 1fr;
+  }
+}
+
+// Small devices (landscape phones, 576px and up)
+@media (min-width: 576px) {
+  .cards {
+    grid-template-columns: 1fr;
+  }
+}
+
+// Medium devices (tablets, 768px and up)
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+// Large devices (desktops, 992px and up)
+@media (min-width: 992px) {
+  .cards {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+// Extra large devices (large desktops, 1200px and up)
+@media (min-width: 1200px) {
+  .cards {
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
Could possibly take off some screen sizes if the number of columns are the same (such as line 31 and line 36)? But we could keep the code like this until we actually have the styled cards to test out responsiveness, it will be easier to see the effects then :)

Reply:
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/113856982/201949661-12d0380e-f7db-4fa4-89e9-9037659dc789.png">

Sure. Regarding the line for head of the layout html, it's already there no?

Reply:
Oh yeah didn't see that sorry!